spec/views/users/index.html.erb_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
```

Comment:
Great job team 🤝
- Some of your tests are still failing the test checks. Kindly ensure all your tests are passing. ![failing-tests](https://github.com/Marcraphael12/rails-blog-app/assets/63917725/d338a96d-348e-4a62-a0aa-9d39282dafc2)

Reply:
![image](https://github.com/Marcraphael12/rails-blog-app/assets/68046127/1d0ae039-17d3-4775-a046-b4c720ffc1b1)